<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>단전 시공서 자동입력기</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>작업할 폴더(*.jpg 파일)</string>
   </property>
   <widget class="QLineEdit" name="LE_OCR_PATH">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_OCR_PATH">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>작업 후 이동할 폴더</string>
   </property>
   <widget class="QLineEdit" name="LE_MOVE_PATH">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_MOVE_PATH">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>자동입력할 엑셀파일</string>
   </property>
   <widget class="QLineEdit" name="LE_EXCEL_PATH">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_EXCEL_PATH">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>찾기</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>상태</string>
   </property>
   <widget class="QPushButton" name="BTN_RUN">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>실행</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PTE_STATUS">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>실행 대기 중...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
